### ISSUE TITLE
Line on home page could confuse users about accessibility of Notify emails

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#1548]
## Describe the bug 
As per [slack thread](https://gcdigital.slack.com/archives/C06KQFM10BU/p1713881480243219)
Content in the Ally statement may conflict with content on home page. May give departments an incorrect impression by stating that emails sent from Notify are accessible:
**Ally statement:** Recipients who use assistive technology may have difficulty with emails from GC Notify
This is because:
- Screen readers do not always read text in the correct language (WCAG 3.1.2 Language of Parts)
- NVDA announces extra spaces in Microsoft Outlook. (I need to figure out which WCAG #)
- Headings within the email start at h2 instead of h1 (WGAG 4.1.2 Name, Role, Value)

**Notify home page states**:
EN: Send consistent and accessible email messages from a trusted domain.
FR: Envoyez des courriels accessibles et cohérents à partir d’un domaine de confiance.

## Expected behavior

Remove "and accessible" in EN and in FR remove "accessibles et" (confirmed with Linguistic services)

## Impact

May give users an incorrect impression that emails sent from Notify are always fully accessible.

**Comment by mtoutloff** [comment]
Gave a head's up to Growth in case this language is recycled in growth/marketing materials
